app/src/main/java/org/mifos/selfserviceapp/ui/activities/LoginActivity.java

```diff
@@ -59,7 +59,7 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     public void onLoginSuccess(String userName) {
         loginStatus = true;
-        showToast(getString(R.string.toast_welcome, userName));
+        showToast(getString(R.string.toast_welcome, userName.replaceFirst("\\s++$", "")));
```

Comment:
@therajanmaurya, I was trying to implement the following - 

The above mentioned code does this, it gets rid of the trailing spaces in the toast message.

![videotogif_2017 03 18_18 16 40](https://cloud.githubusercontent.com/assets/20839661/24072187/152485c0-0c09-11e7-955c-f589391748aa.gif)


If I revert the code, it will look like this - 

![videotogif_2017 03 18_18 26 58](https://cloud.githubusercontent.com/assets/20839661/24072188/152a0b62-0c09-11e7-8f47-25fc7c1a5be3.gif)

Please advise on how to proceed further. Should I still revert the changes ? Also should I implement the above mentioned in a different way. Kindly advise.